Geometry/MuonNumbering/src/ME0NumberingScheme.cc

```diff
@@ -90,8 +96,11 @@ int ME0NumberingScheme::baseNumberToUnitNumber(const MuonBaseNumber& num) {
   else                                    
     region =-1;
   layer   = num.getBaseNo(theLayerLevel)+1;
-  roll=1;
   chamber = num.getBaseNo(theSectorLevel) + 1;
+
+  if(theNEtaPart==1)  roll = 0;
+  if(theNEtaPart==10) roll = num.getBaseNo(theRollLevel)+1;
```

Comment:
i prefer to wait for the comparisons in order to check that the issue you showed in the plot is solved.